future-apps/payroll/package.json

```diff
@@ -18,13 +18,14 @@
   "description": "",
   "devDependencies": {
     "@aragon/apps-vault": "^2.0.1",
-    "@aragon/test-helpers": "^1.0.0",
+    "@aragon/test-helpers": "^1.0.1",
     "babel-helpers": "^6.24.1",
     "babel-polyfill": "^6.26.0",
     "babel-preset-es2015": "^6.18.0",
     "babel-preset-stage-2": "^6.24.1",
     "babel-preset-stage-3": "^6.17.0",
     "babel-register": "^6.26.0",
+    "ethereumjs-abi": "^0.6.4",
```

Comment:
@bingen Is this needed? I remember taking out `ethereumjs-abi` in most of the apps since the test-helpers already includes it.

Reply:
Yes, I saw it's there, but at some point I had an error saying it was missing, and I thought it was weird. I'll double check, just in case.

Reply:
Yes, I tried to remove it again and I got:
```
Error: Cannot find module 'ethereumjs-abi'
    at Function.Module._resolveFilename (module.js:555:15)
    at Function.Module._load (module.js:482:25)
    at Module.require (module.js:604:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/bingen/workspace/aragon-apps/future-apps/payroll/node_modules/@aragon/test-helpers/evmScript.js:1:75)
    at Module._compile (module.js:660:30)
    at Object.Module._extensions..js (module.js:671:10)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
    at Function.Module._load (module.js:505:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @aragon/future-apps-payroll@1.0.0 ganache-cli:test: `./node_modules/@aragon/test-helpers/ganache-cli.sh`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @aragon/future-apps-payroll@1.0.0 ganache-cli:test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

Reply:
Can you try removing your node modules and lock files (e.g. `package-lock.json`), reinstalling, and then trying again?

I think this is usually due to npm ending up in a weird state after uninstalling something.

Reply:
Ok, it seems that did the trick. Thanks! Removing that dependency then.